cmd/apiserver.go

```diff
@@ -31,7 +32,8 @@ func newAPIServerCMD() *cobra.Command {
 		appcatEnabled = viper.GetBool("APPCAT_HANDLER_ENABLED")
 		vshnPGBackupsEnabled = viper.GetBool("VSHN_POSTGRES_BACKUP_HANDLER_ENABLED")
 		vshnRedisBackupsEnabled = viper.GetBool("VSHN_REDIS_BACKUP_HANDLER_ENABLED")
-		if !appcatEnabled && !vshnPGBackupsEnabled && !vshnRedisBackupsEnabled {
+		vshnMariaDBBackupEnabled = viper.GetBool("VSHN_MARIADB_BACKUP_HANDLER_ENABLED")
+		if !appcatEnabled && !vshnPGBackupsEnabled && !vshnRedisBackupsEnabled && !vshnMariaDBBackupEnabled {
```

Comment:
This starts to not scale very well. Maybe we should come up with something better?

Reply:
Let me think about smth here

Reply:
The problem here is that we create the cmd object from the ApiServer instance. After the cmd is created there is no way to add Flags... Let's keep the current implementation and we come up with smth when we move away from the runtime repository.

Reply:
Ah right, we're forced into that config library because of the runtime...